OneCommunityGlobal/HighestGoodNetworkApp
# switch the team member tasks to infinite scroll

# Description
Try to fix the frozen issue of the dashboard page. Using the infinite scroll list to optimize the render. 
Check the [Slack Thread](https://highest-good.slack.com/archives/CSCN20FA5/p1690228096920209) for more details.

## Main changes explained:
- Switch the "Team Member Tasks" to Infinite Scroll List.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin/owner user
4. go to dashboard to see if the page is frozen.

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/62367016/15b8a679-d171-455b-b59a-2cbf4e86c188
